documentation/frontend/guides/quickstart.md

```diff
@@ -39,10 +39,11 @@ you need to run this.
    ```
 
 4. To bring up the frontend, we have another `just` recipe. We have `just`
-   recipes for almost everything.
+   recipes for almost everything. Also, build it with the second command.
```

Comment:
These lines look out of place and don't apply to the change mentioned in this PR. Do you mind removing them?

Reply:
Done mam. If there's anything else please tell me.